app/client/components/viewCommon.css

```
@@ -207,9 +210,11 @@
   border-left-color: var(--grist-theme-table-header-border, var(--grist-color-dark-grey));
 }
 
-.column_name.selected {
-  color: var(--grist-theme-table-header-selected-fg, unset);
-  background-color: var(--grist-theme-table-header-selected-bg, var(--grist-color-medium-grey-opaque));
+.column_name.selected > .column_name_selection {
+  background-color: var(--grist-theme-table-header-selected, var(--grist-color-header-selection));
```

```suggestion
  background-color: var(--grist-theme-selection-header);
```

The following changes will also be needed for this to work:

1. Add the following to `ThemePrefs.ts` in the "Selection" section:
```
'selection-header': string;
```
2. In `GristDark.ts` and `GristLight.ts` respectively, add the following in the same section:
```
'selection-header': 'rgba(107,107,144,0.4)',
```
```
'selection-header': 'rgba(217,217,217,0.6)',
```
3. In `cssVars.ts`, add the following in the same section:
```
selectionHeader: new CustomProp('theme-selection-header', undefined, colors.mediumGrey),
```
4. In `GridView.css`, add the following rule, and add a z-index of 1 to the rules targeting `.g-column-label .info_toggle_icon` and `.g-column-label .kf_editable_label`:
```
.column_name .menu_toggle {
  z-index: 1;
}
```

This should cause the header background color to tint on selection, without  affecting the header text color, and other elements that appear in the header (e.g. menu toggle, description icon).